docs/scos/dev/migration-concepts/migrate-to-paas/step-3-migrate-non-standard-services.md

```diff
@@ -0,0 +1,27 @@
+---
+title: 'Step 3: Migrate non-standard services'
+description: To migrate to PaaS, you need to migrate non-standard services.
+template: howto-guide-template
+---
+
+Spryker comes pre-configured with the following services by default:
+
+* RabbitMQ
+* ElasticSearch
+* Redis
+* MariaDB
+* Jenkins
+
+After you have [upgraded the PHP version](/docs/scos/dev/migration-concepts/migrate-to-paas/step-2-upgrade-the-php-version.html), you can migrate these services without significant challenges. However, make sure to validate their versions. If your project uses components beyond the standard configuration, such as Kafka, we highly recommend conducting a thorough investigation.
```

Comment:
"make sure to validate their versions" - are there any recommendations as to what versions should these services have?
"we highly recommend conducting thorough investigation" - what specific investigation do you mean here?

Reply:
1. Not really. So here we need to check if Spryker would work the same way if on-prem version of e.g. ElasticSearch is different from what Spryker Cloud provides. If the versions are too much different and app is not compatible with what Spryker Cloud offers then we should act toward solving this gap. So far i only found requirements for application, but not for Cloud: https://docs.spryker.com/docs/scos/dev/system-requirements/202311.0/system-requirements.html#base-shop-system-requirements
2. I mean that they should investigate how to solve this gap. For example they use Kafka, but Spryker doesn't offer it in Cloud. Therefore they either should think about replacement of Kafka with RabbitMQ or wire Kafka to Spryker Cloud somehow, e.g. using the VPN.